src/components/search/SearchPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { getClassesData } from '../../util/DataBaseRequests';
+import SearchBar from './SearchBar';
+import SearchResults from './SearchResults';
+import FilterButtons from './FilterContainer';
+
+const SearchPage = () => {
+  const [classes, setClasses] = useState([]);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+  const [search, setSearch] = useState('');
+  const [category, setCategory] = useState('');
+
+  const fetchClasses = async (searchTerm = '', page = 1) => {
```

Comment:
you can move this function in a separate file

Reply:
Good idea! Thank you!